scripts/performance_test/README.md

```diff
@@ -0,0 +1,98 @@
+# MongoDB Physical Data Model Benchmarking Scripts
```

Comment:
This is all good, perhaps we could link to this from the main `CONTRIBUTING.md` file with an overview of the performance test purposes?

Aside: can we rename `performance_test` to `performance-test` as we use snake-case for dir/file names throughout this repo?